docs/user-guide/cli-using-daemon-mode.md

```diff
@@ -1,46 +1,70 @@
 
-**Note:** The Mac executable may need an exception for an unidentified developer: [Open a Mac app from an unidentified developer](https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac)
+   **Note:** Complete the environment variable configuration step (Step 2) only one time.
 
-## Running zowe commands with the zowe executable
+The following example illustrates running Zowe commands with the daemon mode enabled:
 
-Verify that you have installed [the @next release of Zowe CLI](cli-install-cli-next.md) and the [zowe native executable](#install-the-zowex-native-executable).
+   ```
+   zowe --version
+   Starting a background process to increase performance ...
+   7.0.0-next.202110211759
+   
+   zowe --version
+   7.0.0-next.202110211759
+   ```
 
-Run any zowe command as you normally would. The first time you run any zowe command, the command will automatically start a daemon in the background. It will then run your desired command. Since that first command must start the daemon, that first zowe command will actually run slower than a traditional zowe command. However, every zowe command afterward will run significantly faster. The daemon will continue to run in the background until you close your terminal window.
+## Disable daemon mode
 
-Example:
+You can disable Zowe from running in daemon mode at any time. For example, daemon mode lacks functionality that you desire, such as viewing colored-coded commands.
 
-```text
-zowe --version
-Starting a background process to increase performance ...
-7.0.0-next.202110211759
+Open a terminal window and issue the following command:
 
-zowe --version
-7.0.0-next.202110211759
 ```
+zowe daemon disable
+```
+
+The disable command removes the Zowe executables from your .zowe/bin directory and disables daemon mode.
+
+## Running Zowe commands in daemon mode
+
+When you run Zowe in daemon mode, you run all Zowe commands as you normally run them. The first time you run a command, it starts the daemon in the background automatically and runs your desired Zowe command. Since the first Zowe command starts the daemon, the first command usually runs slower than a traditional Zowe command. However, subsequent Zowe commands run significantly faster. The daemon continues to run in the background until you close your terminal window.
+
+If you want to run Zowe in daemon mode constantly, add the path of the $ZOWE_CLI_HOME/bin (.zowe/bin) executables to the directory path of your computer's environment variables. When you configure Zowe to constantly run in daemon mode, daemon mode starts automatically after you issue the first Zowe command. When you open multiple terminal windows and issue Zowe commands, the Zowe commands in all of the terminal windows run in daemon mode.
```

Comment:
@JamesBauman since step 2 under `Enable daemon mode` is required, I think this paragraph can be removed.